README.md

```diff
@@ -32,6 +32,7 @@ In addition to this, the following custom settings are required for this pipelin
 * `FLEXIBLE_OPERATOR`: Global airflow variable which determines the operator that will be used to process the events, the possible values could be `bash` or `kubernetes`. Defaults to `bash`.
 * `pipe_events.events_dataset`: BigQuery dataset which will contain the published events. Defaults to `EVENTS_DATASET`.
 * `pipe_events.source_dataset`: BigQuery dataset which contains the different tables that are read to produce the summarized events. Defaults to `PIPELINE_DATASET`.
+* `pipe_events.publish_to_postgres`: Flag to decide if the results are published to posgres. Defaults to `False`.
```

Comment:
@smpiano mention here that this can be specified at event level instead of `pipe_events` level. Since if you only want to publish ports, encounters and fishing but not gaps you can do it.